src/gui/advancedsettings.cpp

```diff
@@ -453,7 +456,12 @@ void AdvancedSettings::loadAdvancedSettings()
     addRow(HASHING_THREADS, (tr("Hashing threads") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#hashing_threads", "(?)"))
             , &m_spinBoxHashingThreads);
 #endif
-
+	// Max active rechecking torrents
+    m_spinBoxMaxActiveChecking.setMinimum(-1);
```

Comment:
What does -1 mean? Is it some "special" value?

Reply:
Sir it means no limit if I understand correctly. Though can't find anything in documentation.

Reply:
>Sir it means no limit if I understand correctly. 

Then what could 0 mean?
Anyway it is a good idea to use some text/symbol for "special" value in spin box (e.g. `∞`).

Reply:
Ok so I set it to -1 and all started checking at once.
When I set it to 0, all changed their status to checking at 0% and stayed there. So zero means nothing is actively checking. Basically disables or halts the rechecking process. Could be useful if someone mistakely rechecked 100s of torrents, they can pause the rechecking process and restart the client to cancel it altogether.

Reply:
>Anyway it is a good idea to use some text/symbol for "special" value in spin box (e.g. `∞`).

Done.